src/main/java/io/strimzi/kafka/bridge/http/HttpSinkBridgeEndpoint.java

```diff
@@ -67,6 +67,7 @@ public void handle(Endpoint<?> endpoint) {
         // TODO: it seems that getBodyAsJson raises an exception when the body is empty and not null
         try {
             bodyAsJson = routingContext.getBodyAsJson();
+            log.debug("Request: body = {}", bodyAsJson);
```

Comment:
Again, this is find for debugging when you're typically only making a single request at a time, but it needs a correlation id to be useful for analysing production logs.